exotations/solvers/exotica_ompl_control_solver/package.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>exotica_ompl_control_solver</name>
```

Comment:
plural, as above

Reply:
Again, it's `ompl_solver` that we currently have.